website_and_docs/content/documentation/webdriver/elements/finders.en.md

```diff
@@ -108,6 +108,38 @@ driver implementation supports a given feature. These interfaces are clearly def
 to adhere to having only a single role of responsibility.
 {{% /pageinfo %}}
 
+### Evaluating the Shadow DOM
+
+The Shadow DOM is an encapsulated DOM tree hidden inside an element. With the release of v96 in Chromium Browsers, Selenium can now allow you to access this tree with easy-to-use shadow root methods. NOTE: These methods require Selenium 4.0 or greater.
```

Comment:
HI @shbenzer ,

Thank you for the PR.

Could you please limit each line to 100 characters or fewer?
see __Line Length__ in here https://www.selenium.dev/documentation/about/style/

Reply:
My bad, I fixed it.